Doc/library/os.rst

```diff
@@ -1730,6 +1747,8 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    make sense to block because there are no writers connected to the write end
    of the pipe.
 
+   ..seealso:: The :manpage:`splice(2)` man page.
```

Comment:
```suggestion
   .. seealso:: The :manpage:`splice(2)` man page.
```
@aminalaee , thank you for this improvement!
it turns out that rST wants to see a space in the directive, so everything else is fine.